index.html

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Portfolio Design" />
+    <meta name="author" content="Adeyemi Dada" />
+    <link href="assets/css/style.css" rel="stylesheet">
+    <title>Portfolio Design</title>
+</head>
+
+<body>
+    <nav>
+        <ul class="menu">
+            <li class="logo"><a href="#" class="logo_link link text-center fz-18 fw-700 lh-20">My logo</a></li>
+            <li class="item"><a href="#">Portfolio</a></li>
+            <li class="item"><a href="#">About</a></li>
+            <li class="item"><a href="#">Contact</a></li>
+            <li class="mobile-menu"><a href="#">
+                    <img src="assets/images/header-menu.svg" alt="Hamburguer Icon">
+                </a></li>
+        </ul>
+    </nav>
+    <section class="heading flex">
+        <h1 class="heading__title fw-700">I'm Adeyemi.<br /> Glad to see you!</h1>
+        <p class="heading__desc">
+            I’m a software developer! I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact
+            me.
+        </p>
+        <p class="heading__connect">LET’S CONNECT</p>
+        <div class="heading__icons">
+            <ul class="heading__icons__ul ul flex">
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/twitter.svg" alt="twitter icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/linkedin.svg" alt="linkedin icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/medium.svg" alt="github icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/github.svg" alt="github icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/angellist.svg" alt="angellist icon"></a></li>
+            </ul>
+        </div>
+    </section>
```

Comment:
- [ ] Great work implementing changes from the previous reviewer. However, there're few things you still need to work on.
   -  As suggested by the previous reviewer, at breakpoint `768px` which is the starting point for the desktop version of your project does not display this section as expected. Kindly check the screenshot of both designs and walk your design alongside the Figma design.

### Your Design Screenshot
![your-headline-768px](https://user-images.githubusercontent.com/26597432/236408174-7aac6b29-5e13-4a23-a8dd-add0e0bf6b86.png)

### Your Design Screen Recording
[your-app-behaviour.webm](https://user-images.githubusercontent.com/26597432/236408227-3e3c06cb-3229-4f69-a5e2-73359573526b.webm)


### Figma Design
![temp-headline](https://user-images.githubusercontent.com/26597432/236408595-49661157-e94b-442c-88a5-5d60c1880f65.png)